src/index.js

```diff
@@ -29,6 +29,14 @@ const RELEASE_CONFIGS = {
     binaryName: "op-reth",
     getAssetName: (version) => `op-reth-${version}-x86_64-unknown-linux-gnu.tar.gz`,
     fileType: "tar"
+  },
+  "docker-compose": {
```

Comment:
I would prefer to install this not from a binary but from the official docker repo https://github.com/flashbots/builder-playground/blob/main/scripts/ci-setup-docker-compose.sh